action.yml

```diff
@@ -232,15 +242,35 @@ runs:
     shell: bash
 
   - name: Log the next step action
-    run: echo ▷ Figuring out ansible-test flags
+    run: >-
+      echo ▷ Compose the ansible-test command line
     shell: bash
-  - name: Determine ansible-test flags
-    id: ansible-test-flags
+  - name: Compose the ansible-test command line
+    id: ansible-test-command
+    env:
+      GHA_ANSIBLE_CORE_VERSION: ${{ inputs.ansible-core-version }}
+      GHA_CONTAINER_NETWORK: ${{ job.container.network }}
+      GHA_CONTROLLER_PYTHON_VERSION: ${{ inputs.controller-python-version }}
+      GHA_COVERAGE: ${{ inputs.coverage }}
+      GHA_DOCKER_IMAGE: ${{ inputs.docker-image }}
+      GHA_INTEGRATION_CONTINUE_ON_ERROR: >-
+        ${{ inputs.integration-continue-on-error }}
+      GHA_INTEGRATION_DIFF: ${{ inputs.integration-diff }}
+      GHA_INTEGRATION_RETRY_ON_ERROR: ${{ inputs.integration-retry-on-error }}
+      GHA_PULL_REQUEST_BASE_REF: ${{ github.event.pull_request.base.ref || '' }}
```

Comment:
Non-input event contexts should be trusted — no need to reexpose them through env vars, I think. It adds a substantial amount of overhead.

Reply:
I disagree, I think it's better to have pure Python code that doesn't need template evaluation to become an executable script.

Reply:
Yeah, I agree on the point that it's nice to have things layered. Ideally, in the future, Python would move to a dedicated script like in https://github.com/re-actors/alls-green.

I suppose my initial reaction was to many changes being combined (functional/refactoring/etc) which usually adds complexity in perception.

Anyway, I'm marking this as resolved now.